src/sidebar/helpers/view-filter.js

```diff
@@ -53,7 +58,7 @@ function normalize(val) {
 class TermFilter {
   /**
    * @param {string} field - Name of field to match
-   * @param {string} term - Query term
+   * @param {string|number} term - Query term
```

Comment:
I had to go re-read this PR's description to understand why this could be a `number`. Might be worth commenting somewhere.